server/testdriver/core/src/main/java/org/infinispan/server/test/core/ContainerInfinispanServerDriver.java

```diff
@@ -139,7 +139,7 @@ protected void start(String name, File rootDir, String configurationFile) {
             image
                   .withFileFromPath("target", serverOutputPath.getParent())
                   .withFileFromPath("src", serverOutputPath.getParent().getParent().resolve("src"))
-                  .withFileFromPath("build", serverOutputPath);
+                  .withFileFromPath("build", validateServerOutput(serverOutputPath));
```

Comment:
We should cleanup the directories before the call to `image` and we should add a comment explaining why the delete is necessary:

```java
// Removing the `server/data` and `server/log` directories from the local server directory to prevent issues with the tests
Util.recursiveFileRemove(serverOutputPath.resolve("server").resolve("data").toString());
Util.recursiveFileRemove(serverOutputPath.resolve("server").resolve("log").toString());
```